### PR TITLE
Update docs for deployment to add Amazon support for windows

### PR DESCRIPTION
**Pull request by ammarjohar**

### CODE REVIEW

File: end-user/en/source/pages/templating/deploy-machine-image.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Deploying a Machine Image
 --------------------------
 
-.. note:: Currently, only Linux machine images published on Amazon, OpenStack, Cloudstack and Microsoft Azure Resource Manager can be deployed from UForge. You cannot deploy a Windows based appliance.
+.. note:: Currently, Linux machine images published on Amazon, OpenStack, Cloudstack and Microsoft Azure Resource Manager can be deployed from UForge. For Windows based appliances deploy from UForge is supported only for the images published on Amazon.
```
Comment (emuus) [inline end-user/en/source/pages/templating/deploy-machine-image.rst]: For Windows based appliances, (add comma) deploy is supported ...

Comment (ammarjohar) [reply end-user/en/source/pages/templating/deploy-machine-image.rst]: Done.

**Comment by ammarjohar** [comment]
one more note is added kindly verify it.

**Comment by yanngit** [comment]
OK for me too

**Pull request merged**